modules/javafx.graphics/src/main/java/javafx/scene/SubScene.java

```diff
@@ -306,10 +306,9 @@ protected void invalidated() {
                                 "is already set as root of another scene or subScene");
                     }
 
-                    // disabled, isTreeVisible and isTreeShowing properties are inherited
+                    // disabled and isTreeVisible properties are inherited
                     _value.setTreeVisible(isTreeVisible());
                     _value.setDisabled(isDisabled());
-                    _value.setTreeShowing(isTreeShowing());
```

Comment:
This looks fine. It might be worth adding a unit test to ensure that `isTreeShowing` works correctly for nodes in a `SubScene`.

Reply:
I've added a test for this in `TreeShowingExpressionTest`.